src/main/resources/db/migration/v0.25.0.1__disease_annotation_refactor.sql

```diff
@@ -0,0 +1,136 @@
+ALTER TABLE annotation_note_aud RENAME CONSTRAINT fka8x8613e1vmoaxljdrvhftj7a TO annotation_note_aud_rev_fk;
+
+ALTER TABLE diseaseannotation_conditionrelation RENAME TO annotation_conditionrelation;
+ALTER TABLE annotation_conditionrelation RENAME COLUMN diseaseannotation_id TO annotation_id;
+ALTER TABLE annotation_conditionrelation DROP CONSTRAINT fk2wrhxll4ol0fxdpynbmue5stm;
+ALTER TABLE annotation_conditionrelation RENAME CONSTRAINT fklvpswihwk7vf7ijnyustt47na TO annotation_conditionrelation_conditionrelations_id_fk;
+ALTER TABLE annotation_conditionrelation ADD CONSTRAINT annotation_conditionrelation_annotation_id_fk FOREIGN KEY (annotation_id) REFERENCES annotation(id);
+ALTER INDEX idxhnh1h715wjhoulhu2k77ibdfs RENAME TO annotation_conditionrelation_annotation_id_index;
+ALTER INDEX idxtrk7q1phvahyem27wlkmiyw8w RENAME TO annotation_conditionrelation_conditionrelations_id_index;
+
+ALTER TABLE diseaseannotation_conditionrelation_aud RENAME TO annotation_conditionrelation_aud;
+ALTER TABLE annotation_conditionrelation_aud RENAME COLUMN diseaseannotation_id TO annotation_id;
+ALTER TABLE annotation_conditionrelation_aud RENAME CONSTRAINT diseaseannotation_conditionrelation_aud_pkey TO annotation_conditionrelation_aud_pkey;
+ALTER TABLE annotation_conditionrelation_aud RENAME CONSTRAINT fk2i4y7kdvdurhtxhq4qb8ijy0c TO annotation_conditionrelation_aud_rev_fk;
+
+ALTER TABLE annotation
+	ADD COLUMN curie varchar,
+	ADD COLUMN modentityid varchar,
+	ADD COLUMN modinternalid varchar,
+	ADD COLUMN uniqueid varchar (2000),
+	ADD COLUMN dataprovider_id bigint,
+	ADD COLUMN singlereference_curie varchar;
+
+UPDATE annotation SET curie = diseaseannotation.curie FROM diseaseannotation WHERE annotation.id = diseaseannotation.id;
+UPDATE annotation SET modentityid = diseaseannotation.modentityid FROM diseaseannotation WHERE annotation.id = diseaseannotation.id;
+UPDATE annotation SET modinternalid = diseaseannotation.modinternalid FROM diseaseannotation WHERE annotation.id = diseaseannotation.id;
+UPDATE annotation SET uniqueid = diseaseannotation.uniqueid FROM diseaseannotation WHERE annotation.id = diseaseannotation.id;
+UPDATE annotation SET dataprovider_id = diseaseannotation.dataprovider_id FROM diseaseannotation WHERE annotation.id = diseaseannotation.id;
+UPDATE annotation SET singlereference_curie = diseaseannotation.singlereference_curie FROM diseaseannotation WHERE annotation.id = diseaseannotation.id;
+DELETE FROM annotation WHERE dataprovider_id IS NULL;
+
+ALTER TABLE annotation ADD CONSTRAINT annotation_singlereference_curie_fk FOREIGN KEY (singlereference_curie) REFERENCES reference (curie);
+ALTER TABLE annotation ADD CONSTRAINT annotation_dataprovider_id_fk FOREIGN KEY (dataprovider_id) REFERENCES dataprovider (id);
+ALTER TABLE annotation ADD CONSTRAINT annotation_modentityid_key UNIQUE (modentityid);
+ALTER TABLE annotation ADD CONSTRAINT annotation_modinternalid_key UNIQUE (modinternalid);
```

Comment:
typically this would be named: annotation_modinternalid_uk

Reply:
Yeah, that's what I thought too - I was going with the format of some existing UK constraints in the database, but I prefer the _uk suffix so will change to that

Reply:
Changed now